src/metrics_server/README.md

```diff
@@ -67,20 +67,20 @@ npm run action metrics_server/start
   ```
 - To deploy to dev:
   ```sh
-  npm run action metrics_server/deploy_dev
+  task metrics_server:deploy:dev
   ```
 - To deploy to prod:
   ```sh
-  npm run action metrics_server/deploy_prod
+  task metrics_server:deploy:prod
   ```
 
 ## Test
 
 - Unit test
   ```sh
-  npm run action metrics_server/test
+  task metrics_server:test
   ```
 - Integration test
   ```sh
-  npm run action metrics_server/test_integration
+  task metrics_server:integration:test
```

Comment:
*nit* shall we rename it to `integration_test`? Cuz we don't have other sub-tasks for `integration`?